api/src/test/java/com/epam/pipeline/security/saml/SAMLUserDetailsServiceImplTest.java

```diff
@@ -220,6 +220,32 @@ public void shouldThrowAuthorizationExceptionForBlockedUser() {
         userDetailsService.loadUserBySAML(credential);
     }
 
+    @Test
+    public void testBlockedValueCaseNonSensitiveness() {
+        int countExceptionsThrown = 0;
+        for (String blockedValue : SAML_ATTRIBUTE_BLOCKED_USER_VALUES) {
+            try {
+                shouldThrowAuthorizationExceptionForUserWithBlockedApplicationProperty(blockedValue);
+            } catch (LockedException e) {
+                countExceptionsThrown++;
+            }
+        }
+        Assert.assertEquals(SAML_ATTRIBUTE_BLOCKED_USER_VALUES.length, countExceptionsThrown);
+    }
+
+    @Transactional(propagation = Propagation.REQUIRES_NEW, rollbackFor = Exception.class)
+    public void shouldThrowAuthorizationExceptionForUserWithBlockedApplicationProperty(final String blockedValue) {
+        blockUserUsingCredentials(blockedValue);
+        userDetailsService.loadUserBySAML(credential);
+    }
+
+    private void blockUserUsingCredentials(final String blockedValue) {
+        user.setUserName(USER_NAME);
+        Mockito.when(userManager.loadUserByName(Matchers.anyString())).thenReturn(user);
+        ReflectionTestUtils.setField(userDetailsService, "blockedAttribute", blockedValue);
```

Comment:
I would prefer to do not use `blockedValue` variable at this line (to my mind it is not clear). Could you use a new one "test_value" e.g?

Reply:
Done

Reply:
@ekazachkova I'm not sure what change was required here?

Reply:
Sorry, my comment is really confusing.
The idea was just to make test code a little close to real life. It is possible to set `SAMLUserDetailsServiceImpl#blockedAttribute` as "TRUE", but in fact SAML attribute name will be more specific (attribute name and attribute value would not be the same). So it would be more clearly to create a new test constant (but mocked `credential#getAttributeAsString` method should still return boolean value).